src/main/java/hudson/plugins/plot/Plot.java

```diff
@@ -464,20 +474,17 @@ private int getHeight() {
      * directory where the CSV file is located. Unfortunately, a reference to
      * the project is not available when this object is created.
      *
-     * @param project
-     *            the project
+     * @param project the project
      */
-    public void setProject(AbstractProject<?, ?> project) {
+    public void setProject(Job<?, ?> project) {
```

Comment:
🐛  if there are any dependent plugins -- breaking binary method compatibility.  We should create a new version of the setter that takes Job and have this version delegate to it (so the original method can still be invoked).

Reply:
Sounds reasonable, done 👍